keyboards/ckeys/handwire_101/keymaps/default/keymap.c

```diff
@@ -0,0 +1,179 @@
+   * |-----------+-----------+------------+------------|
+   * |     X     |LEFT CLICK |     UP     |RIGHT CLICK |
+   * |-----------+-----------+------------+------------|
+   * |  BUTTON 4 |   LEFT    |    DOWN    |   RIGHT    |
+   * |-----------+-----------+------------+------=-----|
+   * |  BUTTON 3 |SCROLL LEFT|SCROLL DOWN |SCROLL RIGHT|
+   * `-------------------------------------------------'
+   */
+  [_MOUSE] = KEYMAP(
+    KC_MS_BTN5, _____,         KC_MS_WH_UP,   _____,         \
+    _____,      KC_MS_BTN1,    KC_MS_UP,      KC_MS_BTN2,    \
+    KC_MS_BTN4, KC_MS_LEFT,    KC_MS_DOWN,    KC_MS_RIGHT,   \
+    KC_MS_BTN3, KC_MS_WH_LEFT, KC_MS_WH_DOWN, KC_MS_WH_RIGHT \
+  ),
+    /* TERMINAL
+   * ,---------------------------------------.
+   * |            |ABOUT|          |    X    |
+   * |------------+-----+----------+---------|
+   * |TERMINAL OFF|PRINT|          |         |
+   * |------------+-----+----------+---------|
+   * |     X      |FLUSH|          |         |
+   * |------------+-----+----------+---------|
+   * |TERMINAL ON |HELP |          |         |
+   * `--------=======------------------------'
+   */
+  [_TERMINAL] = KEYMAP(
+    _____,    TERM_ABOUT, _____, _____, \
+    TERM_OFF, TERM_PRINT, _____, _____, \
+    _____,    TERM_FLUSH, _____, _____, \
+    TERM_ON,  TERM_HELP , _____, _____ \
+  ),
+    /* ADMIN
+   * ,-----------------------------------------.
+   * |   RESET    |     |           |    X     |
+   * |------------+-----+-----------+----------|
+   * |ABOUT CKEYS |     |           |          |
+   * |------------+-----+-----------+----------|
+   * |            |     |CLICKY UP  |CLICKY OFF|
+   * |------------+-----+-----------+----------|
+   * |     X      |     |CLICKY DOWN|CLICKY ON |
+   * `-----------------------------------------'
+   */
+  [_ADMIN] = KEYMAP(
+    RESET,       _____, _____, _____,  \
+    CKEYS_ABOUT, _____, _____, _____,  \
+    _____,       _____, _____, CK_OFF, \
+    _____,       _____, _____, CK_ON \
+  ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case TERM_ABOUT:
+      if (record->event.pressed) {
+        // when keycode TERM_ABOUT is pressed
+        SEND_STRING("about"SS_TAP(X_ENTER));
+      } else {
+        // when keycode TERM_ABOUT is released
+      }
+      break;
+    case TERM_PRINT:
+      if (record->event.pressed) {
+        SEND_STRING("print"SS_TAP(X_ENTER));
+      } else { }
+      break;
+    case TERM_FLUSH:
+      if (record->event.pressed) {
+        SEND_STRING("flush"SS_TAP(X_ENTER));
+      } else { }
+      break;
+    case TERM_HELP:
+      if (record->event.pressed) {
+        SEND_STRING("help"SS_TAP(X_ENTER));
+      } else { }
+      break;
+    case CKEYS_ABOUT:
+      if (record->event.pressed) {
+        SEND_STRING("https://cKeys.org"SS_TAP(X_ENTER)"Making people smile one keyboard at a time."SS_TAP(X_ENTER)"cKeys is a volunteer run 501(c)(3) nonprofit."SS_TAP(X_ENTER));
```

Comment:
Grammar fix, because I'm *that guy.*

```suggestion
        SEND_STRING("https://cKeys.org"SS_TAP(X_ENTER)"Making people smile one keyboard at a time."SS_TAP(X_ENTER)"cKeys is a volunteer-run 501(c)(3) nonprofit organization."SS_TAP(X_ENTER));
```

Reply:
Haha, thanks for this. I should update my branden@ckeys.org signature now too! 😄